### ISSUE TITLE
CSV response of bulk insertion contains too many headers

### ISSUE THREAD

**Issue by purpleKarrot** [begriffs/postgrest#775]
I want to get list of created ids in csv format:

```
POST /peope?select=id
Accept: text/csv
Content-Type: text/csv
Prefer: return=representation

name,age,height
J Doe,62,70
Jonas,10,55
```

And I get the following back:

```
id,name,age,height
12
13
```

While it really only returns the ids as requested, there are some additional headers.

(I use the begriffs/postgrest docker image)

**Comment by begriffs** [comment]
What are the other response headers that you want to remove?

**Comment by purpleKarrot** [comment]
It is not about http response headers. It is about the CSV headers (the
first line in the response body). I expect it to contain just "id" in this
particular case.

On Wed, Dec 14, 2016, 19:13 Joe Nelson <notifications@github.com> wrote:

What are the other response headers that you want to remove?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/begriffs/postgrest/issues/775#issuecomment-267110016>,
or mute the thread

**Comment by begriffs** [comment]
I see now, totally understand. Yeah that's a bug.

**Comment by steve-chavez** [comment]
This was already fixed by #769.

**Comment by begriffs** [comment]
Nice! We should add a regression test for this particular issue, verify it passes, and then we can close this.

**Comment by begriffs** [comment]
@steve-chavez ...looks like you're busy working on it. Thanks!

**Comment by steve-chavez** [comment]
@begriffs The regression test was included in the #779 merged PR, in https://github.com/begriffs/postgrest/pull/779/commits/4a3852fbd5f5178e724d5b18d19c9b7afcd51154, this issue can be closed.

**Comment by begriffs** [comment]
Oops, I overlooked that.
